module/metrics/gossipsub_rpc_validation_inspector.go

```diff
@@ -53,3 +64,8 @@ func (c *GossipSubRpcValidationInspectorMetrics) AsyncProcessingFinished(duratio
 	c.rpcCtrlMsgInAsyncPreProcessingGauge.Dec()
 	c.rpcCtrlMsgAsyncProcessingTimeHistogram.Observe(duration.Seconds())
 }
+
+// InvalidControlMessageNotificationErrors tracks the number of errors in each invalid control message notification over time.
+func (c *GossipSubRpcValidationInspectorMetrics) InvalidControlMessageNotificationErrors(count int) {
+	c.rpcCtrlMsgAsyncProcessingTimeHistogram.Observe(float64(count))
```

Comment:
❗ this seems wrong, the name is notification errors, but the metric collector is the async processing time.

Reply:
https://github.com/onflow/flow-go/pull/4978/commits/77530d00fece3ba71101a7898a0ae885e5f6caea